BrainCloudClient/src/BrainCloud/Unity/Scripts/BrainCloudLoginPF.cs

```diff
@@ -23,13 +23,17 @@ public class BrainCloudLoginPF : MonoBehaviour
         private Vector2 _scrollPosition;
         private string _authStatus = "Welcome to brainCloud";
 
+        public static BrainCloudWrapper BCWrapper;
+        
         void Start()
         {
             ///////////////////////////////////////////////////////////////////
             // brainCloud game configuration
             ///////////////////////////////////////////////////////////////////
-
-            BrainCloudWrapper.Initialize();
+            
+            
+            BCWrapper = BrainCloudWrapper.GetInstance(); // gameObject.AddComponent<BrainCloudWrapper>();
```

Comment:
BCWrapper = new BrainCloudWrapper() ... why are we using GetInstance() ?

Reply:
so you are making a singleton pattern in the example ? instead of using the newer way to do it ?